_data/people.yml

```diff
@@ -94,7 +94,7 @@
     last_name: Lu
     nick: haonanl
     type: phd
-    active: true
+    active: false
```

Comment:
Do we maybe want a second category for post docs? If so, we can still merge this and I’ll add it separately.

Reply:
I was thinking the same thing. I actually tried to add a post-docs section, but the styles seem to break when there's only one person in a section. I didn't have good enough CSS skills to fix it.